pyam/logging.py

```diff
@@ -6,7 +6,9 @@
 
 
 @contextmanager
-def adjust_log_level(logger, level="ERROR"):
+def adjust_log_level(logger="pyam.core", level="ERROR"):
```

Comment:
I'd rather use "pyam" (ie all pyam modules) as the default rather than just core.